docs/dev/agent_memory.md

```diff
@@ -136,9 +136,40 @@ available in the links below.
 Python, another runtime in the Agent process, is also garbage collected.
 Datadog offers two tools with the Agent that can help you identify memory issues:
 
+- Python memory telemetry (Python 3 only)
 - Trracemalloc
 - Pympler
 
+### Python memory telemetry
+
+Python memory telemetry hooks into low-level allocator routines, to
+provide a coarse view of the total memory allocated by the Python
+memory manager.
+
+Python memory telemetry is only available when using Python 3 (Python
+2 lacks the hooks necessary to implement this).
+
+Python memory telemetry is part of the agent internal telemetry and is
+enabled by default. Set `telemetry.python_memory: false` to disable.
+
+| Internal name  | Default metric name         | Description                                                    |
+|----------------|-----------------------------|----------------------------------------------------------------|
+| `pymem__alloc` | `datadog.agent.pymem.alloc` | Total number of bytes allocated since the start of the agent.  |
+| `pymem__inuse` | `datadog.agent.pymem.inuse` | Number of bytes currently allocated by the python interpreter. |
```

Comment:
```suggestion
| `pymem__alloc` | `datadog.agent.pymem.alloc` | Total number of bytes allocated since the start of the Agent.  |
| `pymem__inuse` | `datadog.agent.pymem.inuse` | Number of bytes currently allocated by the Python interpreter. |
```